rclcpp_lifecycle/test/test_default_state_machine.cpp

```diff
@@ -0,0 +1,205 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
Note to reviewers: I wanted to add and use a `TestLifecycleNode` test fixture. I moved the default_state_machine tests to another file because having two different test fixtures in one file was causing rclcpp::init to be called twice.

Reply:
mmm this file was just renamed... I didn't think we updated copyright in that situation (but I'm not 100% sure about these things)

Reply:
It was my initial fault. It just have been 2016 in the first place :/

Reply:
I removed the test changes completely, it was bloating the diff unnecessarily